tests/integration/fixtures.py

```diff
@@ -5,10 +5,19 @@
 import numpy as np
 import pandas as pd
 import pytest
+from dask.datasets import timeseries
 from dask.distributed import Client
 from pandas.testing import assert_frame_equal
 
 
+@pytest.fixture()
+def training_df(c):
+    df = timeseries(freq="1d").reset_index(drop=True)
+    c.create_table("timeseries", df, persist=True)
```

Comment:
The only problem with this df is, it has no NULL values - and as regr_count only gives non-trivial results in this case...
Could you introduce some NaN values (maybe even randomly)? Maybe 10-30% of the values (independently in x and y, so that we have pairs of both null and only one null and none null)

Reply:
makes perfect sense, I will introduce some NULL values in timeseries_df